### DISCUSSION TITLE
Toggle weather via console command?

### DISCUSSION THREAD

**Discussion by Von-Junzt** [dovrosenberg/foundry-simple-weather#55] [General]
Is there a way to toggle the weather for a scene via the console? Since the last update the toggle button seems to be broken for me. It may be a conflict between modules, can't say it for sure because I couldn't test it yet.

I've got a lot of scenes that are indoors and it's a hassle to always deactivate the weather effects manually, because they will reload after a browser refresh, as long as simple weather is activated for a scene.

**Comment by dovrosenberg** [comment]
I'll take a look - at both fixing the button and adding a key mapping

**Comment by dovrosenberg** [comment]
As far as the broken part, can you share what version of simple weather and Foundry you are using?  Do you have simple calendar installed, too?  If so, which version?  And do you have simple weather docked to simple calendar or in a standalone window?

**Comment by dovrosenberg** [comment]
So, there's actually already a key mapping.  In Foundry "Configure Controls", you can assign a key for "Toggle weather effects" and it will turn them on/off.   Note that if you're using FX Master, it can take a long time for the display to change after you toggle (known issue with FX Master).  You can switch to the core effects to see more easily if it's working, because they turn on/off right away.

I did notice that the toggle button isn't properly displaying the state, so that it can be hard to tell whether they're on or off while FX Master updates.  I'll fix that in the next release.

If you think it's still not working, though, post your version info per my prior comment and I'll dig in more.

**Reply by Von-Junzt** [reply]
Thx, will do!
